keps/0023-enable-disable-features.md

```diff
@@ -0,0 +1,131 @@
+---
+kep-number: 23
+title: Enabling and Disabling features in KUDO Operators
+authors:
+  - "@zmalik"
+owners:
+  - "@zmalik"
+creation-date: 2020-01-28
+last-updated: 2020-01-28
+status: provisional
+---
+
+# Enabling and Disabling features in KUDO Operators
+
+## Summary 
+
+KUDO enables Operator developers to create Operators and develop features for them. 
+The goal of this KEP is to facilitate operator developers to enable and disable the features in a deterministic way.
+
+## Motivation
+
+We want to provide operator users a way to enable and disable features in their operator. And right now there is no operator-user friendly or operator-developer friendly way to do it. 
```

Comment:
nit: _feature_  is an overloaded word. maybe we can take the definition from the L26 and put it here, making it clearer for the reader?

Reply:
L26 had the _example-definition_ I just added a line above `A feature in this KEP is defined as a set of Kubernetes objects.`  WDYT?